src/api/app/models/project.rb

```diff
@@ -89,7 +89,7 @@ def staging_history
   has_many :notified_projects, dependent: :destroy
   has_many :notifications, through: :notified_projects
   has_many :reports, as: :reportable, dependent: :nullify
-  has_many :label_templates
+  has_many :label_templates, dependent: :destroy
   has_many :label_globals
```

Comment:
```suggestion
  has_many :label_globals, dependent: :destroy
```

We should also destroy dependent `label_globals`, I guess... Not tested, tough.

Reply:
No, Global labels are shared between all projects, if I remove a Global label, then I remove it from all the projects. I think that's not what we want.

Reply:
You are right, I always get confused between labels and label templates :facepalm: I'm going to follow-up this one